docs/source/project_documentation/user_guide/model.md

```diff
@@ -0,0 +1,83 @@
+# Model Definitions in AaC
+
+The AaC `model` root key is used to represent the components of a system. These,
+for example, can be the services in a system following a Microservice
+Architecture (MSA) design.
+
+In its most basic form, a `model` must have a `name` that distinguishes it from
+other AaC definitions. At the same time, however, a `model` that defines nothing
+more than a `name` is of little use for anything more than representing external
```

Comment:
At the same time, however, a `model` that defines nothing more than a `name` is only useful for representing external components or actors.

Reply:
I chose not to say "only useful" here because I don't generally like to make absolute statements (Could placeholder models be a valid use case for models with only a name? Could there be other uses that we haven't considered? etc). That said, I slightly modified your suggestion in commit e3672be. Does that work, @lizzcondrey?

Also, on a side note, you can use the "suggestion" button to make a suggestion that will show a diff of what you want changed which makes it a bit easier to see what to change or immediately apply the change.
<img width="441" alt="Screenshot 2023-08-29 at 11 00 11" src="https://github.com/DevOps-MBSE/AaC/assets/2924081/8643acf0-5791-42a4-942c-d8cfcb836067">

Reply:
Yes, that rephrasing works. Better clarification than what I had. 

OMG thank you, haha. For this batch of edits I definitely wanted that ability but doing the review last night I didn't figure out how.